src/genophenocorr/model/_protein.py

```diff
@@ -242,6 +246,18 @@ def motifs(self) -> typing.Iterable[ProteinFeature]:
         """
         return filter(lambda f: f.feature_type == FeatureType.MOTIF, self.protein_features)
 
+    def get_features_variant_overlaps(self, var_start: int, var_end: int) -> typing.Sequence[ProteinFeature]:
+        affected_features = set()
+        for feat in self.protein_features:
+            if feat.info.start is None or feat.info.end is None:
+                print(f"{feat.info.name} has no start and end info")
+                continue
+            if feat.info.start <= var_start <= feat.info.end:
+                affected_features.add(feat.to_string())
+            elif feat.info.start <= var_end <= feat.info.end:
+                affected_features.add(feat.to_string())
+        return affected_features 
```

Comment:
`set` is not a `Sequence`. Moreover, you are not returning a `Set[ProteinFeature]` but a `Set[str]`.